fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -2088,6 +2088,13 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueue* self,
 			healthyDestinations.addReadInFlightToTeam(+metrics.readLoadKSecond());
 
 			launchDest(rd, bestTeams, self->destBusymap);
+			if (doBulkLoading) {
```

Comment:
dataDistributionRelocator is a massive method. A comment on what the (expected) context is when we are at this point would help (I scrolled back a few pages and couldn't figure if this a bulk loading context)

There is an if (doBulkLoading) on line 2060. Could its validateBulkLoadRelocateData be moved here?

Reply:
Added.